selfdrive/car/honda/carstate.py

```diff
@@ -275,18 +302,36 @@ def update(self, cp):
       self.steer_override = abs(cp.vl["STEER_STATUS"]['STEER_TORQUE_SENSOR']) > 1200
     self.steer_torque_driver = cp.vl["STEER_STATUS"]['STEER_TORQUE_SENSOR']
 
-    # brake switch has shown some single time step noise, so only considered when
-    # switch is on for at least 2 consecutive CAN samples
     self.brake_switch = cp.vl["POWERTRAIN_DATA"]['BRAKE_SWITCH']
-    self.brake_pressed = cp.vl["POWERTRAIN_DATA"]['BRAKE_PRESSED'] or \
+    if self.CP.safetyModel == car.CarParams.SafetyModels.hondaBosch:
+      self.stopped = cp.vl["ACC_HUD"]['CRUISE_SPEED'] == 252.
```

Comment:
This probably needs to be renamed to `CRUISE_STATUS` IMO

Reply:
Good point, will talk with @rbiasini about this, naming matches nidec hondas but i think this message matches them

Reply:
Yes, I would read the stopped state from a cruise state msg instead of the hud msg.

Reply:
This is def a weird way to do this, but I can't seem to find another "car stopped" state anywhere else other than that message.

Reply:
I also check `CS.hud_lead == 1.0` but I don't know if that really makes it any better